src/main/java/com/airlines/dao/AirplaneDaoImpl.java

```diff
@@ -0,0 +1,189 @@
+        return Airplane.builder()
+                .withId(resultSet.getInt("id"))
+                .withCodeName(resultSet.getString("codeName"))
+                .withModel(Model.valueOf(resultSet.getString("model")))
+                .withManufacture(resultSet.getDate("manufactureDate").toLocalDate())
+                .withCapacity(resultSet.getInt("capacity"))
+                .withFlightRange(resultSet.getInt("flightRange"))
+                .withCrewId(resultSet.getInt("crewId"))
+                .build();
+    }
+
+    @Override
+    public List<Airplane> findAll() {
+        try (Connection connection = dataSource.getConnection()) {
+            Statement statement = connection.createStatement();
+            ResultSet resultSet = statement.executeQuery(SELECT_ALL_AIRPLANE_SQL);
+            return collectToList(resultSet);
+        } catch (SQLException e) {
+            throw new DaoOperationException(e.getMessage());
+        }
+    }
+
+    private List<Airplane> collectToList(ResultSet resultSet) throws SQLException {
+        List<Airplane> accountList = new ArrayList<>();
+        while (resultSet.next()) {
+            Airplane airplane = parseRow(resultSet);
+            accountList.add(airplane);
+        }
+        return accountList;
+    }
+
+    public boolean deleteAirplane(int id) {
+        boolean rowDeleted;
+        try (Connection connection = dataSource.getConnection()) {
+            PreparedStatement statement = connection.prepareStatement(DELETE_AIRPLANE_SQL);
+            statement.setInt(1, id);
+            rowDeleted = statement.executeUpdate() > 0;
+            return rowDeleted;
+        } catch (SQLException e) {
+            throw new DaoOperationException(e.getMessage());
+        }
+    }
+
+    @Override
+    public List<Airplane> searchAirplanes(String nameCrew) {
+        try (Connection connection = dataSource.getConnection();
+             PreparedStatement preparedStatement = connection.prepareStatement(SELECT_CREW_MEMBERS_BY_CREW_NAME)) {
+            List<Airplane> list = new ArrayList<>();
+            preparedStatement.setString(1, nameCrew);
+            ResultSet resultSet = preparedStatement.executeQuery();
+            while (resultSet.next()) {
+                Airplane airplane = parseRow(resultSet);
+                list.add(airplane);
+            }
+            return list;
+        } catch (SQLException e) {
+            throw new DaoOperationException("Can not perform select Airplane by id query", e);
+        }
+    }
+
+    public void updateAirplane(Airplane airplane, int crews_id) {
+        try (Connection connection = dataSource.getConnection()) {
+            PreparedStatement updateStatement = prepareUpdateStatement(airplane, crews_id, connection);
+            executeUpdate(updateStatement, "Airplane was not updated");
+        } catch (SQLException e) {
+            throw new DaoOperationException(e.getMessage());
```

Comment:
try to use  ("Can not update airplane with id = %s", s), e)

Reply:
fixed in [3d2fa27](https://github.com/vitaly-lab/airlines-app/pull/6/commits/3d2fa27df397dda1795b8b92332e2576be3067c6)